packages/nimble-components/src/utilities/unit-format/tests/decimal-unit-format.spec.ts

```diff
@@ -223,6 +223,88 @@ describe('DecimalUnitFormat', () => {
             });
         });
 
+        describe('normalizeAndDefaultFractionDigitOptions', () => {
+            const normalizeAndDefaultTestCases = [
+                {
+                    name: 'returns min=2 and max=2 when no values are passed',
+                    fractionDigits: undefined,
+                    minimumFractionDigits: undefined,
+                    maximumFractionDigits: undefined,
+                    expectedMinimumFractionDigits: 2,
+                    expectedMaximumFractionDigits: 2
+                },
+                {
+                    name: 'returns min=max=fractionDigits when only fractionDigits passed',
+                    fractionDigits: 7,
+                    minimumFractionDigits: undefined,
+                    maximumFractionDigits: undefined,
+                    expectedMinimumFractionDigits: 7,
+                    expectedMaximumFractionDigits: 7
+                },
+                {
+                    name: 'returns min=0 and max=maximumFractionDigits when only maximumFractionDigits passed',
+                    fractionDigits: undefined,
+                    minimumFractionDigits: undefined,
+                    maximumFractionDigits: 4,
+                    expectedMinimumFractionDigits: 0,
+                    expectedMaximumFractionDigits: 4
+                },
+                {
+                    name: 'returns min=0 and max=undefined when only minimumFractionDigits passed',
+                    fractionDigits: undefined,
+                    minimumFractionDigits: 4,
+                    maximumFractionDigits: undefined,
+                    expectedMinimumFractionDigits: 4,
+                    expectedMaximumFractionDigits: undefined
+                },
+                {
+                    name: 'returns min=minimumFractionDigits and max=maximumFractionDigits when both passed',
+                    fractionDigits: undefined,
+                    minimumFractionDigits: 3,
+                    maximumFractionDigits: 4,
+                    expectedMinimumFractionDigits: 3,
+                    expectedMaximumFractionDigits: 4
+                }
+            ] as const;
+            parameterizeNamedList(
```

Comment:
Note: With #1738 completing, you'll need to update this to `parameterizeSpec` and update the import to be `import { parameterizeSpec } from '@ni/jasmine-parameterized';`